### PR TITLE
Code style cleanup

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

- Remove redundant parentheses
- Property setters should not return anything
- use `if myvar:` instead of `if myvar != []:`

### CODE REVIEW

File: lib/mpl_toolkits/mplot3d/axis3d.py [hunk 1]
```diff
@@ -389,15 +389,15 @@ def d_interval(self):
...
     @d_interval.setter
     def d_interval(self, minmax):
-        return self.set_data_interval(*minmax)
+        self.set_data_interval(*minmax)
 
     @property
     def v_interval(self):
         return self.get_view_interval()
 
-    @d_interval.setter
```
Comment (timhoffm) [inline lib/mpl_toolkits/mplot3d/axis3d.py]: This was actually a bug.

File: lib/matplotlib/dates.py [hunk 2]
```diff
@@ -893,11 +893,11 @@ def __init__(self, locator, tz=None, defaultfmt='%Y-%m-%d'):
...
                        1. / HOURS_PER_DAY: rcParams['date.autoformatter.hour'],
-                       1. / (MINUTES_PER_DAY):
+                       1. / MINUTES_PER_DAY:
                            rcParams['date.autoformatter.minute'],
-                       1. / (SEC_PER_DAY):
+                       1. / SEC_PER_DAY:
                            rcParams['date.autoformatter.second'],
-                       1. / (MUSECONDS_PER_DAY):
+                       1. / MUSECONDS_PER_DAY:
                            rcParams['date.autoformatter.microsecond']}
```
Comment (anntzer) [inline lib/matplotlib/dates.py]: while you're at it this dict could be formatted
```
        self.scaled = {
            DAYS_PER_YEAR: rcParams['date.autoformatter.year'],
            DAYS_PER_MONTH: rcParams['date.autoformatter.month'],
            1: rcParams['date.autoformatter.day'],
            1 / HOURS_PER_DAY: rcParams['date.autoformatter.hour'],
            1 / MINUTES_PER_DAY: rcParams['date.autoformatter.minute'],
            1 / SEC_PER_DAY: rcParams['date.autoformatter.second'],
            1 / MUSECONDS_PER_DAY: rcParams['date.autoformatter.microsecond'],
        }
```

File: lib/matplotlib/mathtext.py [hunk 3]
```diff
@@ -568,7 +568,7 @@ def _get_font(self, font):
 
     def _get_offset(self, font, glyph, fontsize, dpi):
         if font.postscript_name == 'Cmex10':
-            return ((glyph.height/64.0/2.0) + (fontsize/3.0 * dpi/72.0))
```
Comment (anntzer) [inline lib/matplotlib/mathtext.py]: I slightly prefer the spacing `fontsize/3 * dpi/72`.

File: lib/mpl_toolkits/axisartist/axis_artist.py [hunk 4]
```diff
@@ -334,7 +334,7 @@ def draw(self, renderer):
         text_ref_angle = self._get_text_ref_angle()
         offset_ref_angle = self._get_offset_ref_angle()
 
-        theta = (offset_ref_angle)/180.*np.pi
```
Comment (anntzer) [inline lib/mpl_toolkits/axisartist/axis_artist.py]: perhaps rewrite these using np.deg2rad, which has clearer intent (below too)

File: lib/mpl_toolkits/mplot3d/axis3d.py (hunk 1 above)
Comment (dopplershift) [reply lib/mpl_toolkits/mplot3d/axis3d.py]: Would love to have a test for this...

Comment (anntzer) [reply lib/mpl_toolkits/mplot3d/axis3d.py]: We should actually just deprecate them, as they are just mplot3d-specific synonyms for {get,set}_{view,data}_interval...

Comment (timhoffm) [reply lib/mpl_toolkits/mplot3d/axis3d.py]: Moved to #16053.

**Pull request merged**